.docker/ci/Dockerfile

```diff
@@ -29,20 +32,55 @@ RUN \
         wget \
         # Required for rosdep command
         sudo \
-        # Preferred build tools
-        python-catkin-tools \
         clang clang-format-3.9 clang-tidy clang-tools \
-        ccache && \
+        ccache \
+        curl gnupg2 lsb-release \
+        # Some dependencies for working with ROS2
+        build-essential \
+        cmake \
+        git \
+        python3-colcon-common-extensions \
+        python3-pip \
+        python-rosdep \
+        python3-vcstool && \
+    #
+    # Upgrade Pip
+    python3 -m pip install --upgrade pip && \
+    #
+    # install some pip packages needed for testing
+    python3 -m pip install -U \
+        argcomplete \
+        flake8 \
+        flake8-blind-except \
+        flake8-builtins \
+        flake8-class-newline \
+        flake8-comprehensions \
+        flake8-deprecated \
+        flake8-docstrings \
+        flake8-import-order \
+        flake8-quotes \
+        git+https://github.com/lark-parser/lark.git@0.7d \
+        pytest-repeat \
+        pytest-rerunfailures \
+        pytest \
+        pytest-cov \
+        pytest-runner \
+        setuptools && \
+    #
+    # install Fast-RTPS dependencies
+    apt-get install -qq --no-install-recommends -y \
+        libasio-dev \
+        libtinyxml2-dev && \
     #
     # Download all dependencies of MoveIt!
     rosdep update && \
-    rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
+    rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false --skip-keys "console_bridge fastcdr fastrtps libopensplice67 libopensplice69 rti-connext-dds-5.3.1 urdfdom_headers" && \
     #
     # Remove the source code from this container
     # TODO: in the future we may want to keep this here for further optimization of later containers
+    cd $ROS_WS/src && \
```

Comment:
Please merge with next line.

Reply:
I see some of my test code got in to the pr :P

Reply:
fixed